aspnetcore/includes/RP-EF/rp-over-mvc.md

```diff
@@ -1,7 +1,12 @@
-This tutorial teaches ASP.NET Core MVC and Entity Framework Core with controllers and views. Razor Pages is a new alternative in ASP.NET Core 2.0, a page-based programming model that makes building web UI easier and more productive. We recommend the [Razor Pages](xref:data/ef-rp/intro) tutorial over the MVC version. The Razor Pages tutorial:
+This tutorial teaches ASP.NET Core MVC and Entity Framework Core with controllers and views. [Razor Pages](xref:razor-pages/index) is an alternative programming model that was introduced in ASP.NET Core 2.0. For new development, we recommend Razor Pages over MVC with controllers and views. There is a [Razor Pages](xref:data/ef-rp/intro) version of this tutorial. Each tutorial covers some material the other doesn't:
 
-* Is easier to follow.
-* Provides more EF Core best practices.
-* Uses more efficient queries.
-* Is more current with the latest API.
-* Covers more features.
+Some things this MVC tutorial has that the Razor Pages tutorial doesn't:
+
+* Implement inheritance in the data model
+* Perform raw SQL queries
+* Use dynamic LINQ to simplify code
+ 
+Some things the Razor Pages tutorial has that this one doesn't:
+
```

Comment:
* Provides more EF Core best practices.	
 * Uses more efficient queries.	
* Implement inheritance in the data model

Reply:
The EF folks reviewed the RP version (which was a straight port of the MVC version) and made a bunch of changes that never made it back to the MVC version.

Reply:
On best practices and efficient queries, I retrofitted EF MVC with the ones I found while I was updating to 2.2 (e.g., FirstOrDefault).  The only one I'm aware of that I didn't retrofit was Select for related data.  What specifically did I miss?

Reply:
I don't remember, but I thought there were a few more. Go ahead.